packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/Providers/ModelFactories/ModelFactoriesCustomizationTests.cs

```diff
@@ -135,7 +137,9 @@ public async Task CanCustomizeModelFullConstructor()
                             InputFactory.Property("Prop1", InputPrimitiveType.String, isRequired: true),
                         ])
                ],
-               compilation: async () => await Helpers.GetCompilationFromDirectoryAsync());
+               compilation: async () => await Helpers.GetCompilationFromDirectoryAsync(),
+               additionalMetadataReferences: [MetadataReference.CreateFromFile(typeof(BinaryData).Assembly.Location)]);
```

Comment:
why is this needed?

Reply:
This was needed because otherwise the BinaryData symbols are not loaded correctly. It isn't needed for tests in MGC.CM because the ClientModelPlugin already adds the BinaryData reference for us.

Reply:
I see, makes sense. thanks.